packages/web/src/scss/components/Tooltip/_Tooltip.scss

```diff
@@ -85,10 +85,12 @@
 }
 
 // Controlled placement
-.Tooltip[data-spirit-placement-controlled] {
+// stylelint-disable-next-line selector-max-class
+.Tooltip.Tooltip {
```

Comment:
🤔

Reply:
it is just for now, consultated with @crishpeen